csaf-import/src/main/kotlin/io/github/csaf/sbom/Main.kt

```diff
@@ -16,6 +16,30 @@
  */
 package io.github.csaf.sbom
 
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+@KoverIgnore("Entry point for testing purposes only")
 fun main() {
-    println("hello world")
+    val json = Json { prettyPrint = true }
+    val loader = CsafLoader()
+    runBlocking(Dispatchers.IO) {
```

Comment:
I think it would make sense to offer this functionality as an API function. The more I think about it we would probably need some kind of "ParsedAggregator" class that contains
a) the inner data class of Aggregator that describes the JSON
b) additional meta-data, such as when we fetched this, what kind of headers we encountered (etc), because we need this for requirements such as no-redirect
c) a tree of traversed providers, in a data structure similar to that like "ParsedProvider".

Naming subject to change. But we can also do this in a next PR.

Reply:
Tracking this in a separate PR -> https://github.com/csaf-sbom/kotlin-csaf/pull/35

Reply:
Are we still going to include this code for now?

Reply:
Let's keep the main then for now and we will rework it in #35